packages/inter-protocol/src/vaultFactory/liquidation.js

```diff
@@ -30,18 +42,37 @@ const scheduleLiquidationWakeups = async (
   liquidationWaker,
   reschedulerWaker,
 ) => {
-  const [schedules, params] = await Promise.all([
+  const [schedules, params, now] = await Promise.all([
     E(auctioneerPublicFacet).getSchedules(),
     E(auctioneerPublicFacet).getGovernedParams(),
+    E(timer).getCurrentTimestamp(),
   ]);
 
+  // make one observer that will usually ignore the update.
+  observeIteration(subscribeEach(E(auctioneerPublicFacet).getSubscription()), {
+    async updateState(_newState) {
+      if (!cancelToken) {
+        cancelToken = makeCancelToken();
+        void E(timer).setWakeup(now, reschedulerWaker, cancelToken);
+      }
+    },
+  });
+
+  const waitForNewAuctionParams = () => {
+    if (cancelToken) {
+      E(timer).cancel(cancelToken);
+    }
+    cancelToken = undefined;
+  };
+
   trace('SCHEDULE', schedules.nextAuctionSchedule);
   if (!schedules.nextAuctionSchedule?.startTime) {
```

Comment:
Relating to the previous syntax worries, I'm pleased to see that this `?.` was already on master and did not cause any problems. I don't think `?.` is older than `||=`, supporting the suspicion that the latter is safe too. Not suggesting change. Just observing.